sdk/servicebus/service-bus/test/common/environmentVariables.ts

```diff
@@ -0,0 +1,408 @@
+const isNode = !!process && !!process.version && !!process.versions && !!process.versions.node;
+
+export enum Constants {
+  SERVICEBUS_CONNECTION_STRING = "SERVICEBUS_CONNECTION_STRING",
+  QUEUE_NAME = "QUEUE_NAME",
+  QUEUE_NAME_NO_PARTITION = "QUEUE_NAME_NO_PARTITION",
+  QUEUE_NAME_SESSION = "QUEUE_NAME_SESSION",
+  QUEUE_NAME_NO_PARTITION_SESSION = "QUEUE_NAME_NO_PARTITION_SESSION",
+  TOPIC_NAME = "TOPIC_NAME",
+  TOPIC_NAME_NO_PARTITION = "TOPIC_NAME_NO_PARTITION",
+  TOPIC_NAME_SESSION = "TOPIC_NAME_SESSION",
+  TOPIC_NAME_NO_PARTITION_SESSION = "TOPIC_NAME_NO_PARTITION_SESSION",
+  SUBSCRIPTION_NAME = "SUBSCRIPTION_NAME",
+  SUBSCRIPTION_NAME_NO_PARTITION = "SUBSCRIPTION_NAME_NO_PARTITION",
+  SUBSCRIPTION_NAME_SESSION = "SUBSCRIPTION_NAME_SESSION",
+  SUBSCRIPTION_NAME_NO_PARTITION_SESSION = "SUBSCRIPTION_NAME_NO_PARTITION_SESSION",
+  TOPIC_FILTER_NAME = "TOPIC_FILTER_NAME",
+  TOPIC_FILTER_SUBSCRIPTION_NAME = "TOPIC_FILTER_SUBSCRIPTION_NAME",
+  TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME = "TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME",
+  AAD_CLIENT_ID = "AAD_CLIENT_ID",
+  AAD_CLIENT_SECRET = "AAD_CLIENT_SECRET",
+  AAD_TENANT_ID = "AAD_TENANT_ID",
+  RESOURCE_GROUP = "RESOURCE_GROUP",
+  AZURE_SUBSCRIPTION_ID = "AZURE_SUBSCRIPTION_ID",
+  CLEAN_NAMESPACE = "CLEAN_NAMESPACE"
+}
+
+export enum ErrorCode {
+  MISSING_ENV_VAR = "MISSING_ENV_VAR"
+}
+
+const ErrorMessage = {
+  [ErrorCode.MISSING_ENV_VAR]: (envVar: any) =>
+    `Define ${envVar} in your environment before running integration tests.`
+};
+
+function getError(errorCode: ErrorCode, options: any[]): Error {
```

Comment:
Yes, actually I wanted to discuss more about this in person. :/
As we are refactoring the error codes and messages, would it be good to use a model like this?
Perhaps we can have all SDK error codes and messages mapped in a similar way. And then this would be a couple of lines addition to that list of error codes/messages.
This seems inline with what Python SDK had on their docs as well.

Just wanted to put this out so we get a chance to see it and decide if we maybe we can use this in general. 
@AlexGhiondea @bterlson @jonathandturner thoughts?

If not, I'm okay to update to use `getErrorForMissingVariable` like other instances of error message construction.

Reply:
> As we are refactoring the error codes and messages, would it be good to use a model like this? Perhaps we can have all SDK error codes and messages mapped in a similar way. And then this would be a couple of lines addition to that list of error codes/messages.

Right now, the only work in scope for Service Bus in the current milestone and the next is browserification. The next time we touch or make changes to errors thrown by the sdk would be during Track 2 for Service Bus which hasn't been scheduled yet.

> This seems inline with what Python SDK had on their docs as well.

What we saw for Service Bus Python SDK was what is in preview and what hasn't had recent work done in the past few months. They might very well choose a different design when they get to Track 2. Therefore, I wouldn't take that as a reference until they ship a GA or call what they have as Track 2 compliant

> Just wanted to put this out so we get a chance to see it and decide if we maybe we can use this in general.

Its definitely something we can have a design discussion on, but it is also beyond the scope of the work we want to do for browserfication.

Reply:
Got it, updated to use the `getMissingEnvironmentVaribleError` for now as discussed.